internal/test/ouroboros_mock/entry.go

```diff
@@ -109,6 +110,16 @@ var ConversationEntryKeepAliveResponse = ConversationEntry{
 	},
 }
 
+// ConversationEntryKeepAliveResponse is a pre-defined conversation entry for a keep-alive response
+var ConversationEntryKeepAliveResponseWrongResponse = ConversationEntry{
+	Type:       EntryTypeOutput,
+	ProtocolId: keepalive.ProtocolId,
+	IsResponse: true,
+	OutputMessages: []protocol.Message{
+		keepalive.NewMsgKeepAliveResponse(MockKeepAliveWrongCookie),
+	},
+}
```

Comment:
We should add only common conversation entries here. If they are only used in the tests for another package, they should be defined in those tests.